subscriptions.md

```diff
@@ -0,0 +1,304 @@
+# Subscriptions Endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint contains all of the subscription requested
+
+## Main Endpoint
+**GET /api/core/subscriptions**
+
+Retrieve a pageable list of subscriptions. 
+The subscriptions are ordered by eperson number ascending.
```

Comment:
eperson **uuid** ascending (not number) so that all the subscriptions of a specific eperson are guaranteed to be returned consecutively